Tools/generate-manual/DSL/MultiPageDescription.swift

```diff
@@ -18,7 +18,17 @@ struct MultiPageDescription: MDocComponent {
   var body: MDocComponent {
     Section(title: "description") {
       if let discussion = command.discussion {
-        discussion
+        if case let .staticText(text) = discussion {
```

Comment:
Can we factor this body out into a Discussion type and use it the 4 places changed here?

Reply:
I've added a `DiscussionText` struct to the project that encapsulates this logic!